packages/@okta/vuepress-site/docs/guides/custom-email/main/index.md

```diff
@@ -1,8 +1,66 @@
 ---
-title: Customize email templates
+title: Customize email notifications and domains
+excerpt: Learn how to customize and style the default email notifications that Okta sends to end users and customize email domains.
+layout: Guides
 ---
 
-Use these steps to edit a template or to add a template in one of the Okta-supported languages.
+This guide teaches you how to customize and style default Okta email notifications and how to create a custom email domain.
+
+---
+
+**Learning outcomes**
+
+- Customize email notifications.
+- Configure a custom email domain.
+
+**What you need**
+
+An [Okta Developer Edition organization](/signup)
+
+For configuring a custom email domain:
+
+- Access to the DNS records of your public custom domain.
+- An implementation of the [Sender Policy Framework (SPF)](https://tools.ietf.org/html/rfc7208) to prevent sender address forgery. If you already implement SPF in your custom domain, be aware that you must update the SPF record.
+
+**Sample code**
+
+[Using app context to brand an email](#example-of-using-app-context-to-brand-an-email)
```

Comment:
I'd make this into more of a sentence, to provide a little more context, e.g. "See the [Using app context to brand an email] section."